app/util/cleanup_result_dir.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+import os
+
+artifacts_dir = os.environ.get('TAURUS_ARTIFACTS_DIR')
+files_to_remove = ['jmeter.out',
+                   'jmeter-bzt.properties',
+                   'merged.json',
+                   'merged.yml',
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+
+for file in files_to_remove:
+    file_path = Path(artifacts_dir + f'/{file}')
```

Comment:
why not to use `Path(f"{artifacts_dir}/{file}")`

Reply:
Yes your variant looks prettier nice